graylog2-web-interface/src/components/bootstrap/Nav.jsx

```diff
@@ -63,6 +63,28 @@ const Nav = styled(BootstrapNav)(({ theme }) => css`
         }
       }
     }
+    
+    &.nav-sm {
+      > li > a {
+        padding: 8px 12px;
+        font-size: 12px;
+      }
+    };
+
+    &.nav-xs {
+      > li > a {
+        padding: 2px 6px;
+        font-size: 12px;
+      }
+    };
+
+    &.nav-lg {
+      > li > a {
+        padding: 14px 18px;
+        font-size: 18px;
+        line-height: 1.3;
+      }
+    };
```

Comment:
I thought about using the theme font sizes here, but currently we are not using them for e.g. the button component. I think it is more important to unify the different font-sizes for the different bootstrap sizes, than always using the theme styles.